addons/knobs/src/vue/index.js

```diff
@@ -35,3 +41,23 @@ export const vueHandler = (channel, knobStore) => getStory => context => ({
     knobStore.unsubscribe(this.setPaneKnobs);
   },
 });
+
+// "Higher order component" / wrapper style API
+// In 3.3, this will become `withKnobs`, once our decorator API supports it.
```

Comment:
This is 3.3? Should we handle this API migration now?

Reply:
Will delete that file with 4.0.

Let's keep this as backwards compatible as possible.

Reply:
OK then the comment is wrong?

Reply:
Also, I think the solution @tmeasday and I have in mind is backwards-compatible.